history_tab.py

```
@@ -0,0 +1,530 @@
+
+    def populate_data(self, main_data, room_data_list):
+        meter_values = [
+            main_data.get("meter1_reading", "") or "", main_data.get("meter2_reading", "") or "", main_data.get("meter3_reading", "") or ""
+        ]
+        diff_values = [
+            main_data.get("diff1", "") or "", main_data.get("diff2", "") or "", main_data.get("diff3", "") or ""
+        ]
+        
+        if main_data.get("extra_meter_readings"):
+            try: meter_values.extend(json.loads(main_data["extra_meter_readings"]))
+            except: pass # ignore errors
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)